build.sbt

```diff
@@ -15,9 +15,18 @@ ThisBuild / libraryDependencySchemes ++= Seq(
   "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
 )
 
+lazy val javaModuleName = settingKey[String]("Java Module name")
```

Comment:
nit: can you please move this to after the projects are defined? This seems to be too niche and minor to be defined at the top of build.sbt.